frontend/Dockerfile

```
@@ -1,4 +1,4 @@
-FROM node:19.8.1-bullseye-slim as base
+FROM --platform=$TARGETPLATFORM node:19.8.1-bullseye-slim as base
```

```suggestion
FROM node:19.8.1-bullseye-slim as base
```

デフォルトではホストと同じplatformのものを引っ張ってくるはずなので、これは指定しなくて良さそう？